src/scripts/main.js

```diff
@@ -1,7 +1,93 @@
 'use strict';
 
-// Uncomment the next lines to use your game instance in the browser
-// const Game = require('../modules/Game.class');
-// const game = new Game();
+const Game = require('../modules/Game.class');
+const game = new Game();
 
-// Write your code here
+const button = document.querySelector('.button');
+const score = document.querySelector('.game-score');
+
+const rows = document.querySelectorAll('tr');
+
+const messageStart = document.querySelector('.message-start');
+const messageLose = document.querySelector('.message-lose');
+const messageWin = document.querySelector('.message-win');
+
+button.addEventListener('click', () => {
+  if (game.getStatus() === 'idle') {
+    button.innerHTML = 'Restart';
+    button.classList.add('restart');
+    button.classList.remove('start');
+
+    messageStart.classList.add('hidden');
+
+    game.start();
+  } else {
+    button.innerHTML = 'Start';
+    button.classList.add('start');
+    button.classList.remove('restart');
+
+    messageWin.classList.add('hidden');
+    messageLose.classList.add('hidden');
+    messageStart.classList.remove('hidden');
+
+    game.restart();
+  }
+  drawBoard();
+});
+
+document.addEventListener('keydown', (eventKey) => {
```

Comment:
The 'keydown' event listener does not check if the game is over before processing input. Ensure that no moves can be made after the game has ended to prevent unexpected behavior.